pkg/storage/stores/shipper/downloads/table.go

```diff
@@ -77,6 +78,9 @@ func NewTable(name, cacheLocation string, storageClient StorageClient, boltDBInd
 		defer table.dbsMtx.Unlock()
 		defer close(table.ready)
 
+		log, _ := spanlogger.New(spanCtx, "Shipper.DownloadTable")
```

Comment:
can you log inside the init function the number of tables downloaded and their total size after they all return from parallel and then another log for after they are all opened? this way we can see in the trace how much was downloaded, how long it took, and how long opening them took.